duckduckgo/privacy-for-safari
# fix toolbar icon and fix a few warnings

Task/Issue URL: https://app.asana.com/0/1144193759412024/1204184574938009/f
Tech Design URL:
CC:

**Description**:

fix toolbar icon and fix a few warnings

**Steps to test this PR**:
1. Install the Safari extension and enable it
1. Ensure DDG icon shown next to address bar and in the "customise toolbar" view


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
